### ISSUE TITLE
upgrade causes backtrace complaining about 'OCA\DAV\AppInfo\Application' not found 

### ISSUE THREAD

**Issue by jmcclelland** [nextcloud/circles#255]
It seems to be complaining when running the database update named: "Fix the shares to use timestamp instead of datetime"

It happened both during an upgrade in Nextcloud 13 of versions 0.14.1 to version 0.14.4 and again after upgrading to Nextcloud 14 when I went to version 0.15.2.

I'm using postgresql as my database back end and php version 7.0.33 on Debian Stretch.

Here's the backtrace (this is the one from 014.1 to 0.14.4):

```
www-data@lucius:~/nextcloud$ php occ upgrade
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
    0/0 [>---------------------------]   0%
An unhandled exception has been thrown:
Error: Class 'OCA\DAV\AppInfo\Application' not found in /var/lib/nextcloud/apps-local/circles/lib/Service/MiscService.php:207
Stack trace:
#0 /var/lib/nextcloud/apps-local/circles/lib/Service/MiscService.php(192): OCA\Circles\Service\MiscService::getContactData('glockett:63d2fe...')
#1 /var/lib/nextcloud/apps-local/circles/lib/Service/MiscService.php(158): OCA\Circles\Service\MiscService::getDisplayContact('glockett:63d2fe...', 'glockett:63d2fe...', 4)
#2 /var/lib/nextcloud/apps-local/circles/lib/Model/BaseMember.php(143): OCA\Circles\Service\MiscService::getDisplay('glockett:63d2fe...', 4)
#3 /var/lib/nextcloud/apps-local/circles/lib/Model/BaseMember.php(94): OCA\Circles\Model\BaseMember->setUserId('glockett:63d2fe...')
#4 /var/lib/nextcloud/apps-local/circles/lib/Db/MembersRequestBuilder.php(207): OCA\Circles\Model\BaseMember->__construct('glockett:63d2fe...', 4, 'e69945905586cf')
#5 /var/lib/nextcloud/apps-local/circles/lib/Db/MembersRequest.php(125): OCA\Circles\Db\MembersRequestBuilder->parseMembersSelectSql(Array)
#6 /var/lib/nextcloud/apps-local/circles/lib/Migration/RemoveDeadShares.php(99): OCA\Circles\Db\MembersRequest->forceGetAllMembers()
#7 /var/www/nextcloud/lib/private/Repair.php(89): OCA\Circles\Migration\RemoveDeadShares->run(Object(OC\Repair))
#8 /var/www/nextcloud/lib/private/legacy/app.php(966): OC\Repair->run()
#9 /var/www/nextcloud/lib/private/legacy/app.php(903): OC_App::executeRepairSteps('circles', Array)
#10 /var/www/nextcloud/lib/private/Updater.php(365): OC_App::updateApp('circles')
#11 /var/www/nextcloud/lib/private/Updater.php(245): OC\Updater->doAppUpgrade()
www-data@lucius:~/nextcloud$ 
```
Despite the errors, the app seems to be working fine.

**Comment by daita** [comment]
Thanks for your report, should be fixed next release

**Comment by jmcclelland** [comment]
Thanks for the fix! Does this mean I have data that still needs to be upgraded? Or can I ignore these errors and assume the error messages were not relevant?

**Comment by daita** [comment]
every went fine on your setup, error messages were not relevant

**Comment by joshtrichards** [comment]
Fixed in #271
